src/main.rs

```diff
@@ -61,7 +61,8 @@ async fn main() -> Result<(), LemmyError> {
 
   println!(
     "Starting http server at {}:{}",
-    settings.bind, settings.port
+    settings.bind.unwrap(),
```

Comment:
I got a weird error when I tried to do `.unwrap_or_default()` on bind, because `IpAddr` couldn't implement `Default` for some reason.